frontend/src/v5/ui/routes/viewer/noRevisionOverlay/noRevisionOverlay.component.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { CentredContainer } from '@controls/centredContainer';
+import { FormattedMessage } from 'react-intl';
+import { Heading, NoRevisionBackground, Subheading } from './noRevisionOverlay.styles';
+
+type INoRevision = {
+	message: string;
+	isContainer: boolean;
+};
+
+export const NoRevisionOverlay = ({ message, isContainer }: INoRevision) => {
+	const viewType = isContainer ? 'Container' : 'Federation';
+
+	return (
+		<NoRevisionBackground>
+			<CentredContainer>
+				<Heading>
+					<FormattedMessage
+						id="noRevisionOverlay.heading"
+						defaultMessage={`The ${viewType} is empty`}
+					/>
```

Comment:
Every time theres a variable in the formatted message we should use the values prop. If we dont what we are actually doing is changing the defaultMessage of the formatted message and when doing the translation the variable change wont apply
https://formatjs.io/docs/react-intl/components/#usage